File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


def main():

    height = user_prompt()

    for x in range(height):
        print(" "*(height-(x+1))+"#"*(x+1))


def user_prompt():
    while True:
        x = get_int("Enter the height: ")

        if x > 0 and x < 9:
            break

    return x


main()

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
{
int size,width,pointer,x=1;
//Entere the size
do{
    size = get_int("Enter the size: ");
}while(size<=0||size>=9);
width = size+3;
pointer = 0;
for(int i = 0; i<size;i++)
{
    for(int j=0;j<width;j++)
    {
       if(pointer<size-x)
       {
            printf(" ");

       }else if(pointer<size||pointer>=size+2)
       {
          printf("#");
       }else if (pointer>=size||pointer>size+2)
       {
             printf(" ");
       }
       pointer++;
    }
printf("\n");
width++;
x++;
pointer=0;
}



}